csm/core/agent/csm_agent.py

```diff
@@ -232,6 +234,7 @@ def run():
 
 if __name__ == '__main__':
     sys.path.append(os.path.join(os.path.dirname(pathlib.Path(__file__)), '..', '..', '..'))
+    sys.path.append(os.path.join(os.path.dirname(pathlib.Path(os.path.realpath(sys.argv[0]))), '..', '..'))
```

Comment:
One more use of `sys.argv[0]` to define contents of the path list.

Reply:
Comment addressed.